cpp/src/gandiva/gdv_function_stubs.cc

```diff
@@ -794,6 +795,94 @@ const char* gdv_fn_initcap_utf8(int64_t context, const char* data, int32_t data_
   *out_len = out_idx;
   return out;
 }
+
+GANDIVA_EXPORT
+const char* gdv_fn_mask_first_n(int64_t context, const char* data, int32_t data_len,
+                                int32_t n_to_mask, int32_t* out_len) {
+  if (data_len <= 0) {
+    *out_len = 0;
+    return nullptr;
+  }
+
+  int32_t end_idx = data_len < n_to_mask ? data_len : n_to_mask;
+
+  *out_len = data_len;
+
+  char* out = reinterpret_cast<char*>(gdv_fn_context_arena_malloc(context, *out_len));
+  if (out == nullptr) {
+    gdv_fn_context_set_error_msg(context, "Could not allocate memory for output string");
+    *out_len = 0;
+    return nullptr;
+  }
+
+  static char mask_array[] = {
+      '\0', '\0', '\0', '\0', '\0', '\0', '\0', '\0', '\0', '\0', '\0', '\0', '\0', '\0',
```

Comment:
Can you populate this for all 255 values? The input need not be < 128. Also, organise these into 16 values per line so that it becomes easy to read. As of now, this is 14 values per line - you can organise these into 16 values per line in 16 lines

Reply:
When I run the gandiva linter, it breaks down into smaller lines, so I can't make them with 16 values